collab-database/src/fields/type_option/date_type_option.rs

```diff
@@ -445,7 +445,7 @@ impl DateCellData {
     Self {
       timestamp: Some(timestamp),
       end_timestamp: None,
-      include_time: false,
+      include_time: true,
```

Comment:
It would be better to create a new function or pass `include_time`. Because the frontend app also use DateCellData

Reply:
Sure, I'll create a new function since it might conflict with existing frontend usage.